src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,77 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
+  todos: Todo[];
+  selectedTodo: Todo | undefined;
+  onSelect: (todo: Todo) => void;
+}
 
-    <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
+export const TodoList: React.FC<Props> = ({
+  todos,
+  selectedTodo: { id: selectedId } = { selectedId: 0 },
```

Comment:
Using destructuring with default values for `selectedTodo` can lead to unexpected behavior if `selectedTodo` is undefined. Consider handling this case differently to avoid potential issues.